screen_brightness_control/__init__.py

```diff
@@ -233,7 +240,7 @@ def fade_brightness(
 
 
 def list_monitors_info(
-    method: Optional[str] = None, allow_duplicates: bool = False, unsupported: bool = False
+    method: Optional[str] = None, allow_duplicates: bool = ALLOW_DUPLICATES, unsupported: bool = False
```

Comment:
Not sure what this change is for since changing `ALLOW_DUPLICATES` won't change this parameters default value after the library is imported.
```python
>>> FLAG = False
>>> def a(param=FLAG):
...     print(param)
... 
>>> a()
False
>>> FLAG = True
>>> a()
False
```